server/config/swagger.js

```diff
@@ -0,0 +1,23 @@
+import swaggerJSDoc from 'swagger-jsdoc';
+
+// Swagger Definitions
+const swaggerDefinition = {
+  info: {
+    title: 'Author\'s Haven',
+    version: '1.0.0',
+    description: 'A Social platform for the creative at heart',
+  },
+  host: 'kingsmen-ah-backend-staging.herokuapp.com',
```

Comment:
I don't want this hardcoded.

use: `host: process.env.HOST_NAME || 'localhost:3000'`

then modify the `app.json` file to look like this:

```json
{
  "name": "kingsmen-ah-backend",
  "description": "A community of like minded authors that fosters inspiration and innovation by leveraging the modern web.",
  "scripts": {
    "postdeploy": "heroku config:set HOST_NAME=${HEROKU_APP_NAME}.herokuapp.com --app ${HEROKU_APP_NAME} && npm run refreshdb"
  },
  "env": {
    "HEROKU_APP_NAME": {
      "required": true
    },
    "HEROKU_PARENT_APP_NAME": {
      "required": true
    }
  },
  "formation": {},
  "addons": [],
  "buildpacks": [
    {
      "url": "heroku/nodejs"
    }
  ],
  "stack": "heroku-18"
}
```

Reply:
Thankyou @fidelisojeah. 

Here: "heroku config:set HOST_NAME=${HEROKU_APP_NAME}.herokuapp.com --app ${HEROKU_APP_NAME} && npm run refreshdb"
If i am right, this means heroku will check env config for this ${HEROKU_APP_NAME} value